ForumProject/notifications/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from projects.models import Project
+from investors.models import Investor
+from startups.models import Startup
+
+class Notification(models.Model):
+    
+    TRIGGER_CHOICES = [
+        ('follower(s) list changed', 'follower(s) list changed'),
+        ('subscription changed', 'subscription changed'),
+        ('project status changed', 'project status changed'),
+        ('project budget changed', 'project budget changed'),
+        ('message sent', 'message sent'),
+    ]
+    INITIATOR_CHOICES = [
+        ('investor', 'investor'),
+        ('project', 'project'),
+    ]    
+    project = models.ForeignKey(Project, on_delete=models.SET_NULL, related_name='notice_on_project')
+    startup_name = models.CharField(max_length=Startup._meta.get_field('startup_name').max_length)
+    investor = models.ForeignKey(Investor, on_delete=models.SET_NULL, related_name='notice_to_investor')
+    trigger = models.CharField(max_length=50, choices=TRIGGER_CHOICES)
+    initiator = models.CharField(max_length=8, choices=INITIATOR_CHOICES)
+    date_time = models.DateTimeField(auto_now_add=True)
```

Comment:
Consider adding model-level validations or constraints where appropriate. For instance, if a notification should not exist without an associated project or investor under certain conditions, use model constraints to enforce these rules at the database level